ckanext/versioning/fanstatic/versioning.js

```diff
@@ -206,6 +206,7 @@ ckan.module('dataset_versioning_controls', function ($) {
 
         _revert: function (revision_ref, dataset) {
             const action = 'dataset_revert';
+            const body = $('html,body');
```

Comment:
Yeah, it works fine. I'm gonna merge this since, as I have commented. This changes are just temporary for the UX team to have a better understanding on what's going on while they work on the improvement document.